docs/modules/language-tutorial/pages/02_filling_out_a_template.adoc

```diff
@@ -75,22 +75,25 @@ Consider the following example.
 [source,{pkl}]
 .nestedAmends.pkl
 ----
-woodPigeon {
-  name = "Common wood pigeon"
+stockPigeon {
+  name = "Stock pigeon"
   diet = "Seeds"
   taxonomy {
-    species = "Columba palumbus"
+    kingdom = "Animalia"
+    clade = "Columbimorphae"
+    order = "Columbiformes"
+    species = "Columba oenas"
   }
 }
 
-stockPigeon = (woodPigeon) {
-  name = "Stock pigeon"
+woodPigeon {
```

Comment:
This _should_ still amend `stockPigeon`, though!
```suggestion
woodPigeon = (stockPigeon) {
```

Reply:
Oh yeah, you're right - must have overlooked that as I was moving the code around, thank you!